src/js/components/ClickToSelect.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+const METHODS_TO_BIND = [
+  'selectAll'
+];
+
+class ClickToSelect extends React.Component {
+  constructor() {
+    super();
+
+    METHODS_TO_BIND.forEach((method) => {
+      this[method] = this[method].bind(this);
+    });
+  }
+
+  selectAll(event) {
+    event.preventDefault;
+    let selection = document.getSelection();
+    let node = this.target;
+
+    selection.selectAllChildren(node);
```

Comment:
That's fair! I don't feel strongly about it, so whatever you'd prefer is fine with me.